tests/phpunit/tests/feed/atom.php

```diff
@@ -126,6 +157,33 @@ public function test_feed_element() {
 		$this->assertSame( home_url( '/?feed=atom' ), $link[1]['attributes']['href'] );
 	}
 
+	/**
+	 * Test the <feed> element to make sure its present and populated
+	 * with the modified version of the title.
+	 *
+	 * @ticket 13867
+	 */
+	function test_feed_title() {
+		add_filter( 'comments_feed_title', array( $this, 'apply_comments_feed_title' ), 10, 2 );
+		$this->go_to( '?feed=comments-atom' );
+		$feed = $this->do_atom_comments();
+		$xml  = xml_to_array( $feed );
+
+		// Get the feed title.
+		$title = xml_find( $xml, 'feed', 'title' );
+		$this->assertSame( 'Filtered Title', $title[0]['content'] );
+	}
+
+	/**
+	 * Apply the comments feed title filter.
+	 *
+	 * @ticket 13867
+	 */
+	function apply_comments_feed_title( $item_title, $the_title_rss ) {
```

Comment:
```suggestion
	public function apply_comments_feed_title( $item_title, $the_title_rss ) {
```
All methods should indicate visibility. As this method is used in as a callback, it should have `public` visibility.